### PR TITLE
typescript-conditionals  -William Zhang

### PR DESCRIPTION
**Pull request by williamzhang73**
conditional

### CODE REVIEW

File: typescript-conditionals/main.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+};
+console.log('value of isOldEnoughToDrive: ', isOldEnoughToDrive(homer));
+
+function isOldEnoughToDrinkAndDrive(person: Person): boolean {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
```
Comment (shawnkost) [inline typescript-conditionals/main.ts]: This function should only `return false`. It is never okay to drink and drive!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looks good William! Just one small change here

File: typescript-conditionals/main.ts [hunk 2]
```diff
@@ -70,17 +70,14 @@ const homer: Person = {
 };
 console.log('value of isOldEnoughToDrive: ', isOldEnoughToDrive(homer));
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (shawnkost) [inline typescript-conditionals/main.ts]: Do not add `eslint` rules without asking me. We have our `eslint` setup a specific way, so if you are needing to add a rule to bypass an error, then there is something that is going wrong. If you are getting an error, then please reach out and ask, but don't add any rules yourself.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more thing here William!

File: typescript-conditionals/main.ts (hunk 2 above)
Comment (williamzhang73) [reply typescript-conditionals/main.ts]: got it, thanks

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**